src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -2033,19 +2033,19 @@
     <comment>Text label for secondary button the color picker content dialog. When clicked, the operation of selecting a color is cancelled by the user.</comment>
   </data>
   <data name="Profile_CursorColor_NullableColorPicker.NullColorButtonLabel" xml:space="preserve">
-    <value>Use cursor color from color scheme</value>
+    <value>Cursor color from color scheme</value>
     <comment>Label for a button directing the user to use the cursor color defined in the terminal's current color scheme.</comment>
   </data>
   <data name="Profile_Foreground_NullableColorPicker.NullColorButtonLabel" xml:space="preserve">
-    <value>Use foreground color from color scheme</value>
+    <value>Foreground color from color scheme</value>
     <comment>Label for a button directing the user to use the foreground color defined in the terminal's current color scheme.</comment>
   </data>
   <data name="Profile_Background_NullableColorPicker.NullColorButtonLabel" xml:space="preserve">
-    <value>Use background color from color scheme</value>
+    <value>Background color from color scheme</value>
     <comment>Label for a button directing the user to use the background color defined in the terminal's current color scheme.</comment>
   </data>
   <data name="Profile_SelectionBackground_NullableColorPicker.NullColorButtonLabel" xml:space="preserve">
-    <value>Use selection background color from color scheme</value>
+    <value>Selection background color from color scheme</value>

```

Comment:
i wonder... should we shorten all of these to "Use scheme color"? then we only have one?

Reply:
For the record, it would look like this:
![{C997E6F6-1D9A-4F0D-8C21-7A4594BD5D73}](https://github.com/user-attachments/assets/ab86e015-6aa7-4929-a620-21ff751b64ad)

I think that'd be fine, but for anybody that comes along, please share your thoughts 😊